libs/partners/qdrant/langchain_qdrant/fastembed_sparse.py

```diff
@@ -8,7 +10,7 @@ class FastEmbedSparse(SparseEmbeddings):
 
     def __init__(
         self,
-        model_name: str = "Qdrant/bm25",
+        model_name: str = Field(default="Qdrant/bm25", alias="model"),
```

Comment:
This is not a pydantic model -- that's not going to work, so revering changes here